test-data/channels/dummy/linux-64/repodata.json

```diff
@@ -368,6 +368,55 @@
       "track_features": ["   foo   ","  bar  "]
     }
   },
-  "packages.conda": {},
+  "packages.conda": {
+    "foobar-2.0-bla_1.conda": {
+      "build": "bla_1",
+      "build_number": 1,
+      "depends": [
+        "bors <2.0"
+      ],
+      "license": "MIT",
+      "license_family": "MIT",
+      "md5": "bc13aa58e2092bcb0b97c561373d3905",
+      "name": "foobar",
+      "sha256": "97ec377d2ad83dfef1194b7aa31b0c9076194e10d995a6e696c9d07dd782b14a",
+      "size": 414494,
+      "subdir": "linux-64",
+      "timestamp": 1605110689658,
+      "version": "2.0"
+    },
+    "bors-2.1-bla_1.conda": {
+      "build": "bla_1",
+      "build_number": 1,
+      "depends": [
+      ],
+      "license": "MIT",
+      "license_family": "MIT",
+      "md5": "bc13aa58e2092bcb0b97c561373d3905",
+      "name": "bors",
+      "sha256": "97ec377d2ad83dfef1194b7aa31b0c9076194e10d995a6e696c9d07dd782b14a",
+      "size": 414494,
+      "subdir": "linux-64",
+      "timestamp": 1715610974000,
+      "version": "2.1"
+    },
+    "issue_717-2.1-bla_1.conda": {
+      "build": "issue_717",
+      "build_number": 0,
+      "depends": [],
+      "constrains": [
+        "ray[default,data] >=2.9.0,<3.0.0"
```

Comment:
uff

Reply:
Yeah its just a copy of another record but with another extension.